<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="addButton1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>线</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>r</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>30</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>从文件导入</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>130</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>80</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>圆</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="is_Line_R">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>120</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>射线</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton2">
    <property name="geometry">
     <rect>
      <x>730</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>170</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="isLine_L">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>90</y>
      <width>61</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>直线</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>210</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>y2</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="isLine_S">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>150</y>
      <width>51</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>线段</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>180</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>x1</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>491</width>
      <height>441</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="run_button">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>500</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>运行</string>
    </property>
   </widget>
   <widget class="QPushButton" name="del_button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>350</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>删除</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>300</y>
      <width>361</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_button">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>410</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>清空</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>500</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>500</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>交点个数</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="cir_x">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>130</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="cir_y">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>170</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="cir_r">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="line_x1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>180</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="line_x2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>180</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="line_y1">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="line_y2">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>210</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-100000</number>
    </property>
    <property name="maximum">
     <number>100000</number>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
